silo-core/deploy/silo/_siloDeployments.json

```
@@ -1,16 +1,16 @@
 {
   "arbitrum_one": {},
   "optimism": {
-    "UniswapV3-WETH-USDC-Silo": "0x34A40A76823838a22AE42cB5B62F83580900061d"
+    "Silo_UniswapV3-WETH-USDC": "0x34A40A76823838a22AE42cB5B62F83580900061d"
   },
   "sonic": {
-    "S_ETH": "0x9603Af53dC37F4BB6386f358A51a04fA8f599101",
-    "S_USDC.e": "0x4915F6d3C9a7B20CedFc5d3854f2802f30311d13",
-    "S_USDC.e_borrowable_S": "0x062A36Bbe0306c2Fd7aecdf25843291fBAB96AD2",
-    "WETH_USDC.e": "0xcdE453450d830bA284943C3AAb49F19Dc7904b76",
-    "solvBTC.BBN_solvBTC": "0xC1F3d4F5f734d6Dc9E7D4f639EbE489Acd4542ab",
-    "stS_S_Silo": "0x78C246f67c8A6cE03a1d894d4Cf68004Bd55Deea",
-    "wS_scUSD_Silo": "0xFe514E71F0933F63B374056557AED3dBB381C646",
-    "woS_wS": "0x1A030F39a8cf9f0b2649e97cF6d0C7853AeaCf78"
+    "Silo_S_ETH": "0x9603Af53dC37F4BB6386f358A51a04fA8f599101",
```

```suggestion
    "SILO_S:ETH": "0x9603Af53dC37F4BB6386f358A51a04fA8f599101",
```

I'm wondering if this will not looks better

---

It will require more changes as in the current implementation it should match with the filename. I would leave it as it is.